### ISSUE TITLE
[REVIEW]: A database of the healthy human spinal cord morphometry in the PAM50 template space

### ISSUE THREAD

**Issue by roboneuro** [neurolibre/neurolibre-reviews#17]
**Submitting author:** @valosekj (<a href="http://orcid.org/0000-0002-7398-4990">Jan Valošek</a>)
**Repository:** https://github.com/valosekj/PAM50-normalized-metrics-paper
**Branch with paper.md** (empty if default branch): main
**Version:** r20230904
**Editor:** @agahkarakuzu
**Reviewers:** @mathieuboudreau
**Reproducible preprint:** https://preprint.neurolibre.org/10.55458/neurolibre.00017
**Repository archive:** 10.5281/zenodo.10002178
**Data archive:** 10.5281/zenodo.10002180
**Book archive:** 10.5281/zenodo.10002176
**Docker archive:** 10.5281/zenodo.10002182

## Status

[![status](https://neurolibre.org/papers/764a9031ba7a9108dc07a0da1c09dfbb/status.svg)](https://neurolibre.org/papers/764a9031ba7a9108dc07a0da1c09dfbb)

Status badge code:

```
HTML: <a href="https://neurolibre.org/papers/764a9031ba7a9108dc07a0da1c09dfbb"><img src="https://neurolibre.org/papers/764a9031ba7a9108dc07a0da1c09dfbb/status.svg"></a>
Markdown: [![status](https://neurolibre.org/papers/764a9031ba7a9108dc07a0da1c09dfbb/status.svg)](https://neurolibre.org/papers/764a9031ba7a9108dc07a0da1c09dfbb)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/valosekj/PAM50-normalized-metrics-paper/" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@mathieuboudreau, your review will be checklist based. Each of you will have a separate checklist that you should update when carrying out your review.
First of all you need to run this command in a separate comment to create the checklist:

```
@roboneuro generate my checklist
```

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @agahkarakuzu know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

## Checklists

  @mathieuboudreau, please create your checklist typing: `@roboneuro generate my checklist`

**Comment by roboneuro** [comment]
Hello humans, I'm @roboneuro, a robot that can help you with some common editorial tasks.

For a list of things I can do to help you, just type:

```
@roboneuro commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@roboneuro generate pdf
```

**Comment by roboneuro** [comment]
```
Software report:

github.com/AlDanial/cloc v 1.88  T=0.01 s (976.6 files/s, 171163.5 lines/s)
--------------------------------------------------------------------------------
Language                      files          blank        comment           code
--------------------------------------------------------------------------------
TeX                               1             47              8            719
Jupyter Notebook                  1              0            801            104
Markdown                          2             17              0             99
YAML                              5             13             27             86
JSON                              1              0              0              5
Bourne Again Shell                1              0              0              2
--------------------------------------------------------------------------------
SUM:                             11             77            836           1015
--------------------------------------------------------------------------------


gitinspector failed to run statistical information for the repository
```

**Comment by roboneuro** [comment]
Wordcount for `paper.md` is 692

**Comment by roboneuro** [comment]
Failed to discover a `Statement of need` section in paper

**Comment by roboneuro** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- None

MISSING DOIs

- 10.1111/jon.12097 may be a valid DOI for title: Spinal cord normalization in multiple sclerosis
- 10.1212/wnl.0000000000008466 may be a valid DOI for title: Clinically relevant cranio-caudal patterns of cervical cord atrophy evolution in MS
- 10.1038/sj.sc.3101375 may be a valid DOI for title: Changes of cervical spinal cord and cervical spinal canal with age in asymptomatic subjects
- 10.1007/s00234-023-03191-0 may be a valid DOI for title: Association between brain and upper cervical spinal cord atrophy assessed by MRI and disease aggressiveness in amyotrophic lateral sclerosis
- 10.1007/s00234-013-1204-3 may be a valid DOI for title: Brain size and white matter content of cerebrospinal tracts determine the upper cervical cord area: evidence from structural brain MRI
- 10.1002/jmri.20545 may be a valid DOI for title: Upper cervical cord area in early relapsing-remitting multiple sclerosis: cross-sectional study of factors influencing cord size
- 10.1177/2192568220934496 may be a valid DOI for title: The Prevalence of Asymptomatic and Symptomatic Spinal Cord Compression on Magnetic Resonance Imaging: A Systematic Review and Meta-analysis
- 10.1038/s41582-019-0270-5 may be a valid DOI for title: Traumatic and nontraumatic spinal cord injury: pathological insights from neuroimaging
- 10.1136/bmjopen-2017-019809 may be a valid DOI for title: Can microstructural MRI detect subclinical tissue injury in subjects with asymptomatic cervical spinal cord compression? A prospective cohort study
- 10.20944/preprints202202.0355.v1 may be a valid DOI for title: Quantitative MR Markers in Non-Myelopathic Spinal Cord Compression: A Narrative Review
- 10.1016/j.nicl.2021.102680 may be a valid DOI for title: Cervical and thoracic cord atrophy in multiple sclerosis phenotypes: Quantification and correlation with clinical disability
- 10.1016/j.neuroimage.2009.12.121 may be a valid DOI for title: Rapid semi-automatic segmentation of the spinal cord from magnetic resonance images: Application in multiple sclerosis
- 10.1038/s41582-019-0303-0 may be a valid DOI for title: Degenerative cervical myelopathy - update and future directions
- 10.1038/sdata.2016.44 may be a valid DOI for title: The brain imaging data structure, a format for organizing and describing outputs of neuroimaging experiments
- 10.3390/brainsci12081088 may be a valid DOI for title: The Pre-Operative Duration of Symptoms: The Most Important Predictor of Post-Operative Efficacy in Patients with Degenerative Cervical Myelopathy
- 10.1016/j.neuroimage.2017.10.041 may be a valid DOI for title: PAM50: Unbiased multimodal template of the brainstem and spinal cord aligned with the ICBM152 space
- 10.1097/01.bsd.0000181294.67212.79 may be a valid DOI for title: Measurement of the cervical spinal cord volume on MRI
- 10.3389/fneur.2016.00238 may be a valid DOI for title: A Review of the Segmental Diameter of the Healthy Human Spinal Cord
- 10.1371/journal.pone.0118576 may be a valid DOI for title: Age, gender and normalization covariates for spinal cord gray matter and total cross-sectional areas at cervical and thoracic levels: A 2D phase sensitive inversion recovery imaging study
- 10.3389/fnimg.2022.1031253 may be a valid DOI for title: Automatic measure and normalization of spinal cord cross-sectional area using the pontomedullary junction
- 10.3174/ajnr.a5162 may be a valid DOI for title: A novel MRI biomarker of spinal cord white matter injury: T2∗-weighted white matter to gray matter signal intensity ratio
- 10.1111/jon.12666 may be a valid DOI for title: Intersubject Variability and Normalization Strategies for Spinal Cord Total Cross-Sectional and Gray Matter Areas
- 10.1016/j.neuroimage.2016.10.009 may be a valid DOI for title: SCT: Spinal Cord Toolbox, an open-source software for processing spinal cord MRI data
- 10.1097/brs.0000000000001842 may be a valid DOI for title: Prevalence and imaging characteristics of nonmyelopathic and myelopathic spondylotic cervical cord compression
- 10.1002/brb3.797 may be a valid DOI for title: Predictors of symptomatic myelopathy in degenerative cervical spinal cord compression
- 10.3174/ajnr.a5163 may be a valid DOI for title: Clinically Feasible Microstructural MRI to Quantify Cervical Spinal Cord Tissue Injury Using DTI, MT, and T2*-Weighted Imaging: Assessment of Normative Data and Reliability
- 10.1007/s00586-012-2176-4 may be a valid DOI for title: Normal morphology, age-related changes and abnormal findings of the cervical spine. Part II: magnetic resonance imaging of over 1,200 asymptomatic subjects
- 10.1016/j.neuroimage.2010.07.033 may be a valid DOI for title: Unbiased average age-appropriate atlases for pediatric studies
- 10.1111/jon.12372 may be a valid DOI for title: Spinal Cord MR Diffusion Properties in Patients with Degenerative Cervical Cord Compression
- 10.1212/nxg.0000000000000419 may be a valid DOI for title: Heritability of cervical spinal cord structure
- 10.1016/s0513-5117(08)79128-2 may be a valid DOI for title: Acute cervical traumatic spinal cord injury: MR imaging findings correlated with neurologic outcome–prospective study with 100 consecutive patients
- 10.1001/jama.2009.594 may be a valid DOI for title: Gray’s Anatomy: The Anatomical Basis of Clinical Practice
- 10.1002/nbm.3530 may be a valid DOI for title: Tract-specific and age-related variations of the spinal cord microstructure: A multi-parametric MRI study using diffusion tensor imaging (DTI) and inhomogeneous magnetization transfer (ihMT)
- 10.21037/qims-21-782 may be a valid DOI for title: Semi-automated detection of cervical spinal cord compression with the Spinal Cord Toolbox
- 10.3389/fneur.2021.637198 may be a valid DOI for title: Normalization of Spinal Cord Total Cross-Sectional and Gray Matter Areas as Quantified With Radially Sampled Averaged Magnetization Inversion Recovery Acquisitions
- 10.1155/2014/719520 may be a valid DOI for title: Automatic Labeling of Vertebral Levels Using a Robust Template-Based Approach

INVALID DOIs

- None
```

**Comment by agahkarakuzu** [comment]
@valosekj I am working on a solution that will:

1) Let you get rid of all the manual html tags you had to put in markdown (<p style="align-blahblah">). 
2) This will enable adding proper citations, by using the {cite:p}`key` commands. 
3) Then your references will be automatically parsed in the jupyter book, then will be inserted in the PDF. 

I'll let you know today when I confirm proper functionality.

**Comment by roboneuro** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/neurolibre/preprints/neurolibre.00017/neurolibre.00017/10.55458.neurolibre.00017.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/neurolibre/preprints/blob/neurolibre.00017/neurolibre.00017/10.55458.neurolibre.00017.pdf) :page_facing_up: :point_left:

**Comment by agahkarakuzu** [comment]
@valosek currently in your book the citations are in plaintext format, not linked to the bibliography you provided (`paper.bib`). I know this was because I suggested taking the rrsg2020 `index.ipynb` as a reference.

To fix this issue, I have made a few changes in the rrsg2020/paper repository, which can be reflected here for proper use of citations:

1. No more  `<p style="text-align:justify;"></p>`  wrapping around the text. CSS related configs will instead live in `content/_static/custom.css` with the following content:

```css
p {
  text-align: justify;
}
a {
  color: #870000!important;
}
a:hover {
  color: brick!important;
}
a:visited {
  color: #6f42c1!important;
}
body, h1, h2, h3, h4, h5 {
    font-family: 'STIX Two Text';
}
h1 {
    color: #342727!important;
}
.caption {
    text-align:justify;
    line-height:1.25;
    font-size:80%
}

#site-navigation {
    display: none;
}

#main-content{
    margin: 0 auto;
}
```


2. Make jupyter book build use his custom `css` file and the `paper.bib` (instead of `references.bib` by updating the `content/_config.yml`: 

```yml
sphinx:
  recursive_update          : false # A boolean indicating whether to overwrite the Sphinx config (true) or recursively update
    bibtex_reference_style: author_year
    bibtex_default_style: plain
    bibtex_tooltips: true
    html_static_path: ['_static']
    html_css_files: ['custom.css']

bibtex_bibfiles:
  - ../paper.bib
```

3. Update your `index.ipynb` to use {cite:p}`bibtex-key` citation directives instead of plain text. You can take a look at the [latest RRSG notebook](https://github.com/rrsg2020/paper/blob/main/content/index.ipynb) to see how to manage successive citations or `(abc et al. 2010)` (use `{cite:p}`) vs abc et. al. (2010) formatting (use `{cite:t}`).

Eventually, it should look like this: https://preview.neurolibre.org/book-artifacts/rrsg2020/github.com/paper/c0d7479f87cb0ef959ce87ba3a2bffe2ab44c330/_build/_page/index/singlehtml/

I am working on the last bit to automatically populate the PDF with the citations in context. 

Thank you for your patience! If you don't have it in you to deal with any of these, happy to send you a PR.

**Comment by valosekj** [comment]
> 1. No more  `<p style="text-align:justify;"></p>` wrapping around the text. CSS related configs will instead live in [content/_static/custom.css](https://github.com/rrsg2020/paper/blob/main/content/_static/custom.css) with the following content:
>
> 2. Make jupyter book build use this custom css file and the paper.bib (instead of references.bib by updating the content/_config.yml:

Done in https://github.com/valosekj/PAM50-normalized-metrics-paper/commit/c0cc15b61f1413636954fe4af13abaab694d41be and https://github.com/valosekj/PAM50-normalized-metrics-paper/commit/4b1827fa2cd6d776b7426c71ccc730dc852f6e14.

> 3. Update your index.ipynb to use {cite:p}bibtex-key citation directives instead of plain text. You can take a look at the [latest RRSG notebook](https://github.com/rrsg2020/paper/blob/main/content/index.ipynb) to see how to manage successive citations or (abc et al. 2010) (use {cite:p}) vs abc et. al. (2010) formatting (use {cite:t}).

Okay! I tried the following syntax in my [content/index.ipynb](https://github.com/valosekj/PAM50-normalized-metrics-paper/blob/main/content/index.ipynb):

```
{cite:p}(Losseff et al., 1996; Mina et al., 2021; Rocca et al., 2019)
```

but it doesn't seem to be working:

<img width="600" alt="image" src="https://github.com/neurolibre/neurolibre-reviews/assets/39456460/d7ee0ae7-09a4-46ce-8022-310614c934da">

**Comment by agahkarakuzu** [comment]
You should use the citation keys from the bibtex file. 

Let say for John Doe et al. 2017, the respective bibtex entry is

article{doe2017-xy,
Blah, 
Blah, 
}

Then in the text you {cite:p}'doe2017-xy' (quotes are gonna be the funky ones, can't find them on my phone). 

There are many examples in the rrsg notebook. You can use bibdesk to load and navigate your paper.bib, and to copy/paste cite commands.

**Comment by valosekj** [comment]
Do you have any ideas why there is extra white space between the tables 2, 3, and 4 and their captions?

<img width="1027" alt="image" src="https://github.com/neurolibre/neurolibre-reviews/assets/39456460/bce927d0-e9b4-4234-9229-15bf34bc6b60">

Table 1 is rendered properly:

<img width="960" alt="image" src="https://github.com/neurolibre/neurolibre-reviews/assets/39456460/232af9a0-b90f-4cff-bc25-1061947cb4e1">

Code for all tables is pretty similar; see [here](https://github.com/spinalcordtoolbox/PAM50-normalized-metrics/blob/main/notebooks/tables.py).

**Comment by roboneuro** [comment]
&#9899;  **Book Build (Preview)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-06 05:54:16 PDT

**Comment by roboneuro** [comment]
:confetti_ball::confetti_ball::confetti_ball: 
 Congratulations to the authors on constructing the neurolibre reproducible preprint (NRP)! They've certainly stirred some upright excitement by building a database of healthy human spinal cord morphometry, making "spinal-tific" data open-source for all! 
 :hibiscus: Take a loot at the [latest version of your NRP](https://preview.neurolibre.org/book-artifacts/valosekj/github.com/PAM50-normalized-metrics-paper/f447654f129f56d3759cb2e4947b232b04e38007/_build/_page/index/singlehtml/)!

**Comment by valosekj** [comment]
Regarding my [previous comment](https://github.com/neurolibre/neurolibre-reviews/issues/17#issuecomment-1747451024) about the white space below the tables, it should now be resolved by this [recent commit](https://github.com/spinalcordtoolbox/PAM50-normalized-metrics/commit/dd7d3706bdb71b08665f2c9323b007c05159f1aa). Can you rebuild the book one more time?

**Comment by agahkarakuzu** [comment]
@roboneuro build book

**Comment by roboneuro** [comment]
:eyes: Looks like the reproducible preprint you requested has been already built: 
```
[{"time_added":"2023-10-06 12:59:08 +0000","book_url":"https://preview.neurolibre.org/book-artifacts/valosekj/github.com/PAM50-normalized-metrics-paper/f447654f129f56d3759cb2e4947b232b04e38007/_build/_page/index/singlehtml/","download_url":null}]
```

**Comment by agahkarakuzu** [comment]
@valosekj the book is already built at the latest commit, which includes the one you mention. You may want to try figure layout margins, `fig.update_layout(margin=dict(b=0))`.

**Comment by agahkarakuzu** [comment]
@roboneuro build book

**Comment by roboneuro** [comment]
&#9899;  **Book Build (Preview)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-06 07:49:30 PDT

**Comment by roboneuro** [comment]
:confetti_ball::confetti_ball::confetti_ball: 
 Congratulations to the authors for constructing a neurolibre reproducible preprint (NRP) that's a spinal sensation! Your open-source database on healthy human spinal cord morphometry will surely be a backbone for researchers seeking normative values in the PAM50 template space. 
 :hibiscus: Take a loot at the [latest version of your NRP](https://preview.neurolibre.org/book-artifacts/valosekj/github.com/PAM50-normalized-metrics-paper/7c91e5aa8a8e57eb4136608d63edb3d1e85606b7/_build/_page/index/singlehtml/)!

**Comment by agahkarakuzu** [comment]
Ok great! Looks like the issue has been resolved.

**Comment by agahkarakuzu** [comment]
@roboneuro build extended pdf

**Comment by roboneuro** [comment]
&#9899;  **Extended PDF - Build draft** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-06 08:17:39 PDT

**Comment by agahkarakuzu** [comment]
@valosekj the extended pdf builds, but I noticed a small typo there, I'll fix it, then you can follow up.

**Comment by valosekj** [comment]
I quickly checked the [extended PDF](https://preprint.neurolibre.org/10.55458/draft/00017/paper.pdf) and found the following issues:

Page 2 - references are formatted differently: 

<img width="600" alt="image" src="https://github.com/neurolibre/neurolibre-reviews/assets/39456460/7bc1b05d-122d-4cef-99ec-4b7eee79ff5b">

Page 3 - some references are broken:

<img width="600" alt="image" src="https://github.com/neurolibre/neurolibre-reviews/assets/39456460/cc871567-d07a-4d13-8574-4a3441ab3efb">

Page 6 - whole section "3. RESULTS" is missing

**Comment by agahkarakuzu** [comment]
@valosekj re `karakuzu2022-xq`, fixed on the server side. Re Dupre et al. (2022), no issues there, it is another style for inline citation. 

I've just sent a PR to fix issue with Soltrand et al. citations. 

The whole section being missing is expected, because only paragraphs including citations are included in the PDF, for content registration purposes only. We make this clearer now:

![image](https://github.com/neurolibre/neurolibre-reviews/assets/9632322/0d51865e-e9f7-4667-a20d-c420fff0be73)

**Comment by valosekj** [comment]
Thanks! I merged your [PR](https://github.com/valosekj/PAM50-normalized-metrics-paper/pull/2).

Let me know once the final build is done. I will go through it thoroughly one more time.

**Comment by agahkarakuzu** [comment]
@roboneuro build extended pdf

**Comment by roboneuro** [comment]
&#9899;  **Extended PDF - Build draft** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-10 15:59:47 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro build book

**Comment by roboneuro** [comment]
&#9899;  **Book Build (Preview)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-10 15:59:58 PDT

**Comment by roboneuro** [comment]
&#128209; Extended PDF has been compiled! 
                     
 ### For the submitting author 
                     
 1. 👀  Please review the [extended PDF](https://preprint.neurolibre.org/10.55458/draft/00017/paper.pdf) and verify that all references are accurately included. If everything is correct, please proceed to the next steps. **If not, please make the necessary adjustments in the source documents.**                     
 2. ⬇️ [Download the updated `paper.md`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.md). 
                     
 3. ⬇️ [Download the updated `paper.bib`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.bib). 
                     
 4. ℹ️ Please read and confirm the following: 
                     
 > [!IMPORTANT]                     
 > We have added a note in the extended PDF to inform the readers that the narrative content from your notebook content has been automatically added to credit the referenced sources. This note includes citations to the articles                     explaining the [NeuroLibre workflow](https://doi.org/10.31219/osf.io/h89js), [integrated research objects](https://doi.org/10.1371/journal.pcbi.1009651), and the Canadian Open Neuroscience Platform ([CONP](https://conp.ca)). _If you prefer not to include them, please remove the respective citation directives                     in the updated `paper.md` before pushing the file to your repository._                     
                     
 - [ ] I, the submitting author, confirm that I have read the note above.                     
 5. ♻️ Update the respective files in [your source repository](payload['repository_url']) with the files you just downloaded and inform the screener.                     
 ### For the technical screener                     
 Once the submitting author has updated the repository with the `paper.md` and `paper.bib`, please confirm that the PDF successfully builds using the `@roboneuro generate pdf`. 
                     
 :warning: However, DO NOT issue  `@roboneuro build extended pdf` command after the submitting author has updated the `paper.md` and `paper.bib`.

**Comment by agahkarakuzu** [comment]
I noticed a few more issues, will send another PR.

**Comment by roboneuro** [comment]
:robot::speech_balloon::confetti_ball::rocket: 
 Congratulations to the authors for constructing a neurolibre reproducible preprint that ensures our "spinal cord" of knowledge remains flexible and unbound, providing "open-source" insights into the "morphometric measures" and "normative values" that keep us standing tall! 

 :hibiscus: Take a loot at the [latest version of your NRP](https://preview.neurolibre.org/book-artifacts/valosekj/github.com/PAM50-normalized-metrics-paper/58335b930d4836d38d3039b3699d14053c16cbb9/_build/_page/index/singlehtml/)! :hibiscus: 
 --- 
 > [!IMPORTANT] 
 > Please make sure the figures are displayed correctly, code cells are collapsible, and that BinderHub execution is successful.

**Comment by agahkarakuzu** [comment]
@roboneuro build extended pdf

**Comment by roboneuro** [comment]
&#9899;  **Extended PDF - Build draft** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-11 08:23:09 PDT

**Comment by agahkarakuzu** [comment]
@valosekj issues with references appear to be fixed. Can you please follow the steps in the previous message by roboneuro and give it a quick proofread while adding the new paper.md?

**Comment by valosekj** [comment]
I found one more problematic ref:

<img width="600" alt="image" src="https://github.com/neurolibre/neurolibre-reviews/assets/39456460/69b8ee46-655a-41d0-9917-6e67d2279f4e">

I fixed it in https://github.com/valosekj/PAM50-normalized-metrics-paper/commit/4a1022da1cd27d97416501d012d4c33162595a7c.
I also removed one unused ref in https://github.com/valosekj/PAM50-normalized-metrics-paper/commit/b80cbc0afa6e17f89f1d5519e6e4b9dcaa120e5a.

**Comment by agahkarakuzu** [comment]
@roboneuro build extended pdf

**Comment by roboneuro** [comment]
&#9899;  **Extended PDF - Build draft** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-11 11:02:13 PDT

**Comment by roboneuro** [comment]
&#128209; Extended PDF has been compiled! 
                     
 ### For the submitting author 
                     
 1. 👀  Please review the [extended PDF](https://preprint.neurolibre.org/10.55458/draft/00017/paper.pdf) and verify that all references are accurately included. If everything is correct, please proceed to the next steps. **If not, please make the necessary adjustments in the source documents.**                     
 2. ⬇️ [Download the updated `paper.md`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.md). 
                     
 3. ⬇️ [Download the updated `paper.bib`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.bib). 
                     
 4. ℹ️ Please read and confirm the following: 
                     
 > [!IMPORTANT]                     
 > We have added a note in the extended PDF to inform the readers that the narrative content from your notebook content has been automatically added to credit the referenced sources. This note includes citations to the articles                     explaining the [NeuroLibre workflow](https://doi.org/10.31219/osf.io/h89js), [integrated research objects](https://doi.org/10.1371/journal.pcbi.1009651), and the Canadian Open Neuroscience Platform ([CONP](https://conp.ca)). _If you prefer not to include them, please remove the respective citation directives                     in the updated `paper.md` before pushing the file to your repository._                     
                     
 - [ ] I, the submitting author, confirm that I have read the note above.                     
 5. ♻️ Update the respective files in [your source repository](payload['repository_url']) with the files you just downloaded and inform the screener.                     
 ### For the technical screener                     
 Once the submitting author has updated the repository with the `paper.md` and `paper.bib`, please confirm that the PDF successfully builds using the `@roboneuro generate pdf`. 
                     
 :warning: However, DO NOT issue  `@roboneuro build extended pdf` command after the submitting author has updated the `paper.md` and `paper.bib`.

**Comment by agahkarakuzu** [comment]
@roboneuro build extended pdf

**Comment by roboneuro** [comment]
&#9899;  **Extended PDF - Build draft** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-11 11:20:58 PDT

**Comment by roboneuro** [comment]
&#128209; Extended PDF has been compiled! 
                     
 ### For the submitting author 
                     
 1. 👀  Please review the [extended PDF](https://preprint.neurolibre.org/10.55458/draft/00017/paper.pdf) and verify that all references are accurately included. If everything is correct, please proceed to the next steps. **If not, please make the necessary adjustments in the source documents.**                     
 2. ⬇️ [Download the updated `paper.md`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.md). 
                     
 3. ⬇️ [Download the updated `paper.bib`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.bib). 
                     
 4. ℹ️ Please read and confirm the following: 
                     
 > [!IMPORTANT]                     
 > We have added a note in the extended PDF to inform the readers that the narrative content from your notebook content has been automatically added to credit the referenced sources. This note includes citations to the articles                     explaining the [NeuroLibre workflow](https://doi.org/10.31219/osf.io/h89js), [integrated research objects](https://doi.org/10.1371/journal.pcbi.1009651), and the Canadian Open Neuroscience Platform ([CONP](https://conp.ca)). _If you prefer not to include them, please remove the respective citation directives                     in the updated `paper.md` before pushing the file to your repository._                     
                     
 - [ ] I, the submitting author, confirm that I have read the note above.                     
 5. ♻️ Update the respective files in [your source repository](payload['repository_url']) with the files you just downloaded and inform the screener.                     
 ### For the technical screener                     
 Once the submitting author has updated the repository with the `paper.md` and `paper.bib`, please confirm that the PDF successfully builds using the `@roboneuro generate pdf`. 
                     
 :warning: However, DO NOT issue  `@roboneuro build extended pdf` command after the submitting author has updated the `paper.md` and `paper.bib`.

**Comment by agahkarakuzu** [comment]
@roboneuro build book

**Comment by roboneuro** [comment]
&#9899;  **Book Build (Preview)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-11 11:25:33 PDT

**Comment by roboneuro** [comment]
:robot::speech_balloon::confetti_ball::rocket: 
 Congratulations to the authors on creating a neurolibre reproducible preprint (NRP) that's got us all "spinal-ling" with excitement! Your open-source database of healthy human spinal cord morphometry in the PAM50 template space is simply "nerve-rackingly" amazing, providing normative values that will surely keep researchers in good shape! 

 :hibiscus: Take a loot at the [latest version of your NRP](https://preview.neurolibre.org/book-artifacts/valosekj/github.com/PAM50-normalized-metrics-paper/41fc3d7505f4cebe13afff122425889d3e950a10/_build/_page/index/singlehtml/)! :hibiscus: 
 --- 
 > [!IMPORTANT] 
 > Please make sure the figures are displayed correctly, code cells are collapsible, and that BinderHub execution is successful.

**Comment by jcohenadad** [comment]
Beautiful! Where can we modify the manuscript? I noticed some typos

**Comment by agahkarakuzu** [comment]
The quickest option would be revising the `content/index.ipynb` at https://github.com/valosekj/PAM50-normalized-metrics-paper.

P.S. I will integrate spell-check workflows to deal with typos automatically.

**Comment by jcohenadad** [comment]
WIP: https://github.com/valosekj/PAM50-normalized-metrics-paper/pull/4

**Comment by agahkarakuzu** [comment]
@roboneuro build extended pdf

**Comment by roboneuro** [comment]
&#9899;  **Extended PDF - Build draft** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 08:04:05 PDT

**Comment by roboneuro** [comment]
&#128209; Extended PDF has been compiled! 
                     
 ### For the submitting author 
                     
 1. 👀  Please review the [extended PDF](https://preprint.neurolibre.org/10.55458/draft/00017/paper.pdf) and verify that all references are accurately included. If everything is correct, please proceed to the next steps. **If not, please make the necessary adjustments in the source documents.**                     
 2. ⬇️ [Download the updated `paper.md`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.md). 
                     
 3. ⬇️ [Download the updated `paper.bib`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.bib). 
                     
 4. ℹ️ Please read and confirm the following: 
                     
 > [!IMPORTANT]                     
 > We have added a note in the extended PDF to inform the readers that the narrative content from your notebook content has been automatically added to credit the referenced sources. This note includes citations to the articles                     explaining the [NeuroLibre workflow](https://doi.org/10.31219/osf.io/h89js), [integrated research objects](https://doi.org/10.1371/journal.pcbi.1009651), and the Canadian Open Neuroscience Platform ([CONP](https://conp.ca)). _If you prefer not to include them, please remove the respective citation directives                     in the updated `paper.md` before pushing the file to your repository._                     
                     
 - [ ] I, the submitting author, confirm that I have read the note above.                     
 5. ♻️ Update the respective files in [your source repository](payload['repository_url']) with the files you just downloaded and inform the screener.                     
 ### For the technical screener                     
 Once the submitting author has updated the repository with the `paper.md` and `paper.bib`, please confirm that the PDF successfully builds using the `@roboneuro generate pdf`. 
                     
 :warning: However, DO NOT issue  `@roboneuro build extended pdf` command after the submitting author has updated the `paper.md` and `paper.bib`.

**Comment by agahkarakuzu** [comment]
@valosekj can you review this and confirm that it is ok?

**Comment by agahkarakuzu** [comment]
@roboneuro build extended pdf

**Comment by roboneuro** [comment]
&#9899;  **Extended PDF - Build draft** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 14:17:18 PDT

**Comment by roboneuro** [comment]
&#128209; Extended PDF has been compiled! 
                     
 ### For the submitting author 
                     
 1. 👀  Please review the [extended PDF](https://preprint.neurolibre.org/10.55458/draft/00017/paper.pdf) and verify that all references are accurately included. If everything is correct, please proceed to the next steps. **If not, please make the necessary adjustments in the source documents.**                     
 2. ⬇️ [Download the updated `paper.md`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.md). 
                     
 3. ⬇️ [Download the updated `paper.bib`](https://preprint.neurolibre.org/10.55458/draft/00017/paper.bib). 
                     
 4. ℹ️ Please read and confirm the following: 
                     
 > [!IMPORTANT]                     
 > We have added a note in the extended PDF to inform the readers that the narrative content from your notebook content has been automatically added to credit the referenced sources. This note includes citations to the articles                     explaining the [NeuroLibre workflow](https://doi.org/10.31219/osf.io/h89js), [integrated research objects](https://doi.org/10.1371/journal.pcbi.1009651), and the Canadian Open Neuroscience Platform ([CONP](https://conp.ca)). _If you prefer not to include them, please remove the respective citation directives                     in the updated `paper.md` before pushing the file to your repository._                     
                     
 - [ ] I, the submitting author, confirm that I have read the note above.                     
 5. ♻️ Update the respective files in [your source repository](payload['repository_url']) with the files you just downloaded and inform the screener.                     
 ### For the technical screener                     
 Once the submitting author has updated the repository with the `paper.md` and `paper.bib`, please confirm that the PDF successfully builds using the `@roboneuro generate pdf`. 
                     
 :warning: However, DO NOT issue  `@roboneuro build extended pdf` command after the submitting author has updated the `paper.md` and `paper.bib`.

**Comment by agahkarakuzu** [comment]
@roboneuro generate pdf

**Comment by agahkarakuzu** [comment]
@roboneuro build book

**Comment by roboneuro** [comment]
&#9899;  **Book Build (Preview)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 14:47:43 PDT

**Comment by roboneuro** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/neurolibre/preprints/neurolibre.00017/neurolibre.00017/10.55458.neurolibre.00017.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/neurolibre/preprints/blob/neurolibre.00017/neurolibre.00017/10.55458.neurolibre.00017.pdf) :page_facing_up: :point_left:

**Comment by roboneuro** [comment]
:robot::speech_balloon::confetti_ball::rocket: 
 Congratulations to the authors on their neurolibre reproducible preprint! With their open-source database of spinal cord morphometry, they've proven that sharing is indeed caring for a healthy backbone of scientific progress. 

 :hibiscus: Take a loot at the [latest version of your NRP](https://preview.neurolibre.org/book-artifacts/valosekj/github.com/PAM50-normalized-metrics-paper/1efe31795082282b4f4c458053bf5ac1e7b474a1/_build/_page/index/singlehtml/)! :hibiscus: 
 --- 
 > [!IMPORTANT] 
 > Please make sure the figures are displayed correctly, code cells are collapsible, and that BinderHub execution is successful.

**Comment by agahkarakuzu** [comment]
@roboneuro commands

**Comment by roboneuro** [comment]
Hello @agahkarakuzu, here are the things you can ask me to do:

```

# List all available commands
@roboneuro commands

# Add to this issue's reviewers list
@roboneuro add @username as reviewer

# Remove from this issue's reviewers list
@roboneuro remove @username from reviewers

# Get a list of all editors's GitHub handles
@roboneuro list editors

# Assign a user as the editor of this submission
@roboneuro assign @username as editor

# Remove the editor assigned to this submission
@roboneuro remove editor

# Remind an author, a reviewer or the editor to return to a review after a 
# certain period of time (supported units days and weeks)
@roboneuro remind @reviewer in 2 weeks

# Check the references of the PDF for missing DOIs
@roboneuro check references

# Perform checks on the repository
@roboneuro check repository

# Adds a checklist for the reviewer using this command
@roboneuro generate my checklist

# Set a value for version
@roboneuro set v1.0.0 as version

# Set a value for branch
@roboneuro set neurolibre-paper as branch

# Set a value for repository
@roboneuro set https://github.com/organization/repo as repository

# Set a value for the archive DOI
@roboneuro set set 10.5281/zenodo.6861996 as archive

# Set a value for the (data/book/repository/docker) archive DOI.
@roboneuro set 10.5281/zenodo.6861996 as data archive

# Set executable book URI for the preprint.
@roboneuro set book uri

# Mention the EiCs for the correct track
@roboneuro ping track-eic

# Reject paper
@roboneuro reject

# Withdraw paper
@roboneuro withdraw

# Invite an editor to edit a submission (sending them an email)
@roboneuro invite @(.*) as editor

# Generates the pdf paper
@roboneuro generate pdf

# Recommends the submission for acceptance
@roboneuro recommend-accept

# Accept and publish the paper
@roboneuro accept

# Update data on an accepted/published paper
@roboneuro reaccept

# Generates a LaTeX preprint file
@roboneuro generate preprint

# Flag submission with questionable scope
@roboneuro query scope

# Get a link to the complete list of reviewers
@roboneuro list reviewers

# Creates a post-review checklist with editor and authors tasks
@roboneuro create post-review checklist

# Open the review issue
@roboneuro start review

# Check the status of the NeuroLibre preview server.
@roboneuro preview server status

# Check the status of the NeuroLibre preprint (production) server.
@roboneuro preprint server status

# Build a NeuroLibre reproducible preprint for technical screening.
@roboneuro build book

# After screening, build book from the forked reository.
@roboneuro production build book

# Transfer data from preview to the preprint (production) server.
@roboneuro production sync data

# Transfer the final preprint (forked repo) to the preprint (production) server (DOI URL).
@roboneuro production sync book

# Start the production process by forking repository and configuring it.
@roboneuro production start

# Creates Zenodo deposits (a.k.a buckets) for archiving reproducibility assets.
@roboneuro zenodo create buckets

# Status check for the archival of the reproducibility assets.
@roboneuro zenodo status

# Copy summary PDF from the preprints repository to the NeuroLibre production server.
@roboneuro production sync pdf

# Upload the latest Jupyter Book to Zenodo for archival.
@roboneuro zenodo upload book

# Upload the latest version of the book repository to Zenodo for archival.
@roboneuro zenodo upload repository

# Upload the latest version of the docker image to Zenodo for archival.
@roboneuro zenodo upload docker

# Upload the submission data to Zenodo for archival. Will not take effect if the data is already archived.
@roboneuro zenodo upload data

# DESTRUCTIVE ACTION: Deletes zenodo records and all the data that has been uploaded. Cannot perform after zenodo publish.
@roboneuro zenodo flush

# Publish reproducibility assets on Zenodo. Sets DOIs if successful.
@roboneuro zenodo publish

# Build an extended PDF for the target repository.
@roboneuro build extended pdf

```

**Comment by agahkarakuzu** [comment]
@roboneuro production sync book

**Comment by agahkarakuzu** [comment]
@roboneuro preprint server status

**Comment by roboneuro** [comment]
&#128994; NeuroLibre server is active (running). <br> &#127808; Ready to accept requests from Issue #17 <br> &#128279; URL: https://preprint.neurolibre.org

**Comment by agahkarakuzu** [comment]
@roboneuro production sync book

**Comment by agahkarakuzu** [comment]
@roboneuro production sync data

**Comment by roboneuro** [comment]
&#9899;  **DATA TRANSFER (Preview --> Preprint)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 15:03:16 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro production start

**Comment by roboneuro** [comment]
&#9899;  **INITIATE PRODUCTION (Fork and Configure)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 15:03:48 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro production build book

**Comment by roboneuro** [comment]
&#9899;  **Book Build (Preview)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 15:05:19 PDT

**Comment by roboneuro** [comment]
:robot::speech_balloon::confetti_ball::rocket: 
 Congratulations to the authors for creating a neurolibre reproducible preprint that delves deep into the open "spinal" of knowledge, revealing the hidden "cord" of normative values in a format that's sure to keep scientists in "template" overload! 

 :hibiscus: Take a loot at the [latest version of your NRP](https://preview.neurolibre.org/book-artifacts/roboneurolibre/github.com/PAM50-normalized-metrics-paper/5694fa641d21ca90cab508fcec5ccbd3a07ec860/_build/_page/index/singlehtml/)! :hibiscus: 
 --- 
 > [!IMPORTANT] 
 > Please make sure the figures are displayed correctly, code cells are collapsible, and that BinderHub execution is successful.

**Comment by agahkarakuzu** [comment]
@roboneuro production sync book

**Comment by roboneuro** [comment]
&#9899;  **REPRODUCIBLE PREPRINT TRANSFER (Preview --> Preprint)** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 15:15:25 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo create buckets

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Create Zenodo buckets** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 15:41:10 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo create buckets

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Create Zenodo buckets** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 16:21:22 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo create buckets

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Create Zenodo buckets** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 16:25:05 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload data

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Data** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 16:26:51 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload book

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Jupyter Book** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 16:27:00 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload repository

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive GitHub Repository** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 16:27:12 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 16:28:34 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 16:40:47 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 16:52:23 PDT

**Comment by agahkarakuzu** [comment]
@valosekj zenodo is going through a major change and have some upload issues:

![image](https://github.com/neurolibre/neurolibre-reviews/assets/9632322/9d3fc2be-6c82-4437-8a04-bbcd98bb4ec3)

I'll try it with intervals, but right now I am receiving `ConnectionResetError(104, 'Connection reset by peer')` from their servers.

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo status

**Comment by roboneuro** [comment]
<h3>Deposit</h3>:green_square: Zenodo deposit records are found.<h3>Upload</h3><ul><li>:green_circle: Repository archive <ul><li><code>2.01 MB</code> <code>zenodo_uploaded_repository_NeuroLibre_00017_5694fa.json</code></li></ul></li><li>:green_circle: Data archive <ul><li><code>5.04 MB</code> <code>zenodo_uploaded_data_NeuroLibre_00017_5694fa.json</code></li></ul></li><li>:green_circle: Book archive <ul><li><code>13.24 MB</code> <code>zenodo_uploaded_book_NeuroLibre_00017_5694fa.json</code></li></ul></li><li>:red_circle: <b>Docker archive is missing</b></li></ul><h3>Publish</h3>:small_red_triangle_down: <b>Zenodo DOIs have not been published yet.</b>

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-13 17:49:46 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-14 06:24:39 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-15 14:35:27 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 06:00:33 PDT

**Comment by agahkarakuzu** [comment]
<img width="983" alt="image" src="https://github.com/neurolibre/neurolibre-reviews/assets/9632322/25c11863-6656-4b9a-a92f-48bab89a4cda">

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 08:17:50 PDT

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 08:27:02 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 08:39:52 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 08:49:56 PDT

**Comment by agahkarakuzu** [comment]
OK this returns error 413 (content too large) for a `613 MB` of a docker image, which is not exceeding any limits whatsoever. Zenodo issue is still going on.

**Comment by jcohenadad** [comment]
thank you for looking into this @agahkarakuzu ❤️

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 10:44:22 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo upload docker

**Comment by roboneuro** [comment]
&#9899;  **Reproducibility Assets - Archive Docker Image** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 17:00:58 PDT

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo status

**Comment by roboneuro** [comment]
<h3>Deposit</h3>:green_square: Zenodo deposit records are found.<h3>Upload</h3><ul><li>:green_circle: Repository archive <ul><li><code>2.01 MB</code> <code>zenodo_uploaded_repository_NeuroLibre_00017_5694fa.json</code></li></ul></li><li>:green_circle: Data archive <ul><li><code>5.04 MB</code> <code>zenodo_uploaded_data_NeuroLibre_00017_5694fa.json</code></li></ul></li><li>:green_circle: Book archive <ul><li><code>13.24 MB</code> <code>zenodo_uploaded_book_NeuroLibre_00017_5694fa.json</code></li></ul></li><li>:green_circle: Docker archive <ul><li><code>685.10 MB</code> <code>zenodo_uploaded_docker_NeuroLibre_00017_5694fa.json</code></li></ul></li></ul><h3>Publish</h3>:small_red_triangle_down: <b>Zenodo DOIs have not been published yet.</b>

**Comment by agahkarakuzu** [comment]
@roboneuro recommend-accept

**Comment by roboneuro** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by roboneuro** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1111/jon.12097 is OK
- 10.1212/wnl.0000000000008466 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.1038/sj.sc.3101375 is OK
- 10.1007/s00234-023-03191-0 is OK
- 10.1007/s00234-013-1204-3 is OK
- 10.1093/brain/119.3.701 is OK
- 10.1002/jmri.20545 is OK
- 10.1177/2192568220934496 is OK
- 10.1038/s41596-021-00588-0 is OK
- 10.1038/s41597-021-01044-0 is OK
- 10.1038/s41582-019-0270-5 is OK
- 10.1136/bmjopen-2017-019809 is OK
- 10.1016/j.nicl.2021.102680 is OK
- 10.1016/j.neuroimage.2018.09.081 is OK
- 10.3174/ajnr.a5427 is OK
- 10.1016/j.neuroimage.2009.12.121 is OK
- 10.1038/s41582-019-0303-0 is OK
- 10.1038/sdata.2016.44 is OK
- 10.3390/brainsci12081088 is OK
- 10.1016/j.neuroimage.2017.10.041 is OK
- 10.1097/01.bsd.0000181294.67212.79 is OK
- 10.3389/fneur.2016.00238 is OK
- 10.1371/journal.pone.0118576 is OK
- 10.3389/fnimg.2022.1031253 is OK
- 10.3174/ajnr.a5162 is OK
- 10.1111/jon.12666 is OK
- 10.1016/j.neuroimage.2016.10.009 is OK
- 10.1097/brs.0000000000001842 is OK
- 10.1002/brb3.797 is OK
- 10.3174/ajnr.a5163 is OK
- 10.1007/s00586-012-2176-4 is OK
- 10.1016/j.neuroimage.2010.07.033 is OK
- 10.1111/jon.12372 is OK
- 10.1212/nxg.0000000000000401 is OK
- 10.1148/radiol.2433060583 is OK
- 10.1002/nbm.3530 is OK
- 10.21037/qims-21-782 is OK
- 10.3389/fneur.2021.637198 is OK
- 10.1097/00007632-199903150-00023 is OK
- 10.1155/2014/719520 is OK
- 10.1038/s41597-022-01571-4 is OK
- 10.5281/zenodo.6511596 is OK
- 10.31219/osf.io/h89js is OK
- 10.1371/journal.pcbi.1009651 is OK
- 10.1371/journal.pcbi.1011230 is OK

MISSING DOIs

- 10.1001/jama.2009.594 may be a valid DOI for title: Gray’s Anatomy: The Anatomical Basis of Clinical Practice

INVALID DOIs

- None
```

**Comment by roboneuro** [comment]
:wave: @openjournals/executable-eics, this paper is ready to be accepted and published.

Check final proof :point_right::page_facing_up: [Download article](https://raw.githubusercontent.com/neurolibre/preprints/neurolibre.00017/neurolibre.00017/10.55458.neurolibre.00017.pdf)

If the paper PDF and the deposit XML files look good in https://github.com/neurolibre/preprints/pull/56, then you can now move forward with accepting the submission by compiling again with the command `@editorialbot accept`

**Comment by agahkarakuzu** [comment]
@roboneuro set book uri

**Comment by roboneuro** [comment]
Done, set the URI for the reproducible preprint: [https://preprint.neurolibre.org/10.55458/neurolibre.00017](https://preprint.neurolibre.org/10.55458/neurolibre.00017)

**Comment by roboneuro** [comment]
:page_with_curl: Looks like the summary PDF is not available on the NeuroLibre servers yet.

**Comment by roboneuro** [comment]
&#9899;  **Publish Reproducibility Assets** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 17:49:10 PDT

**Comment by agahkarakuzu** [comment]
I'll just test out a feature below.

**Comment by agahkarakuzu** [comment]
@roboneuro zenodo publish

**Comment by roboneuro** [comment]
&#9899;  **Publish Reproducibility Assets** 
 ---------------------------- 
  **Status:** Waiting for task assignment 
 **Last updated:** 2023-10-16 18:13:13 PDT

**Comment by roboneuro** [comment]
@roboneuro set https://doi.org/10.5281/zenodo.10002176 as book archive

**Comment by roboneuro** [comment]
@roboneuro set https://doi.org/10.5281/zenodo.10002178 as repository archive

**Comment by roboneuro** [comment]
@roboneuro set https://doi.org/10.5281/zenodo.10002180 as data archive

**Comment by roboneuro** [comment]
@roboneuro set https://doi.org/10.5281/zenodo.10002182 as docker archive

**Comment by agahkarakuzu** [comment]
Cool, now this monologue works, but as it is not registered as an editor, the commands will not take effect. Sorry for spamming you all.

**Comment by agahkarakuzu** [comment]
@roboneuro set 10.5281/zenodo.10002176 as book archive

**Comment by roboneuro** [comment]
Done, book archive is now [10.5281/zenodo.10002176](https://doi.org/10.5281/zenodo.10002176)

**Comment by agahkarakuzu** [comment]
@roboneuro set 10.5281/zenodo.10002178 as repository archive

**Comment by roboneuro** [comment]
Done, repository archive is now [10.5281/zenodo.10002178](https://doi.org/10.5281/zenodo.10002178)

**Comment by agahkarakuzu** [comment]
@roboneuro set zenodo.10002180 as data archive

**Comment by roboneuro** [comment]
Done, data archive is now [zenodo.10002180](https://doi.org/zenodo.10002180)

**Comment by agahkarakuzu** [comment]
@roboneuro set 10.5281/zenodo.10002182 as docker archive

**Comment by roboneuro** [comment]
Done, docker archive is now [10.5281/zenodo.10002182](https://doi.org/10.5281/zenodo.10002182)

**Comment by agahkarakuzu** [comment]
@roboneuro recommend-accept

**Comment by roboneuro** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by roboneuro** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1111/jon.12097 is OK
- 10.1212/wnl.0000000000008466 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.1038/sj.sc.3101375 is OK
- 10.1007/s00234-023-03191-0 is OK
- 10.1007/s00234-013-1204-3 is OK
- 10.1093/brain/119.3.701 is OK
- 10.1002/jmri.20545 is OK
- 10.1177/2192568220934496 is OK
- 10.1038/s41596-021-00588-0 is OK
- 10.1038/s41597-021-01044-0 is OK
- 10.1038/s41582-019-0270-5 is OK
- 10.1136/bmjopen-2017-019809 is OK
- 10.1016/j.nicl.2021.102680 is OK
- 10.1016/j.neuroimage.2018.09.081 is OK
- 10.3174/ajnr.a5427 is OK
- 10.1016/j.neuroimage.2009.12.121 is OK
- 10.1038/s41582-019-0303-0 is OK
- 10.1038/sdata.2016.44 is OK
- 10.3390/brainsci12081088 is OK
- 10.1016/j.neuroimage.2017.10.041 is OK
- 10.1097/01.bsd.0000181294.67212.79 is OK
- 10.3389/fneur.2016.00238 is OK
- 10.1371/journal.pone.0118576 is OK
- 10.3389/fnimg.2022.1031253 is OK
- 10.3174/ajnr.a5162 is OK
- 10.1111/jon.12666 is OK
- 10.1016/j.neuroimage.2016.10.009 is OK
- 10.1097/brs.0000000000001842 is OK
- 10.1002/brb3.797 is OK
- 10.3174/ajnr.a5163 is OK
- 10.1007/s00586-012-2176-4 is OK
- 10.1016/j.neuroimage.2010.07.033 is OK
- 10.1111/jon.12372 is OK
- 10.1212/nxg.0000000000000401 is OK
- 10.1148/radiol.2433060583 is OK
- 10.1002/nbm.3530 is OK
- 10.21037/qims-21-782 is OK
- 10.3389/fneur.2021.637198 is OK
- 10.1097/00007632-199903150-00023 is OK
- 10.1155/2014/719520 is OK
- 10.1038/s41597-022-01571-4 is OK
- 10.5281/zenodo.6511596 is OK
- 10.31219/osf.io/h89js is OK
- 10.1371/journal.pcbi.1009651 is OK
- 10.1371/journal.pcbi.1011230 is OK

MISSING DOIs

- 10.1001/jama.2009.594 may be a valid DOI for title: Gray’s Anatomy: The Anatomical Basis of Clinical Practice

INVALID DOIs

- None
```

**Comment by roboneuro** [comment]
:wave: @openjournals/executable-eics, this paper is ready to be accepted and published.

Check final proof :point_right::page_facing_up: [Download article](https://raw.githubusercontent.com/neurolibre/preprints/neurolibre.00017/neurolibre.00017/10.55458.neurolibre.00017.pdf)

If the paper PDF and the deposit XML files look good in https://github.com/neurolibre/preprints/pull/57, then you can now move forward with accepting the submission by compiling again with the command `@editorialbot accept`

**Comment by agahkarakuzu** [comment]
@roboneuro recommend-accept

**Comment by roboneuro** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by roboneuro** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1111/jon.12097 is OK
- 10.1212/wnl.0000000000008466 is OK
- 10.1038/s41592-019-0686-2 is OK
- 10.1038/sj.sc.3101375 is OK
- 10.1007/s00234-023-03191-0 is OK
- 10.1007/s00234-013-1204-3 is OK
- 10.1093/brain/119.3.701 is OK
- 10.1002/jmri.20545 is OK
- 10.1177/2192568220934496 is OK
- 10.1038/s41596-021-00588-0 is OK
- 10.1038/s41597-021-01044-0 is OK
- 10.1038/s41582-019-0270-5 is OK
- 10.1136/bmjopen-2017-019809 is OK
- 10.1016/j.nicl.2021.102680 is OK
- 10.1016/j.neuroimage.2018.09.081 is OK
- 10.3174/ajnr.a5427 is OK
- 10.1016/j.neuroimage.2009.12.121 is OK
- 10.1038/s41582-019-0303-0 is OK
- 10.1038/sdata.2016.44 is OK
- 10.3390/brainsci12081088 is OK
- 10.1016/j.neuroimage.2017.10.041 is OK
- 10.1097/01.bsd.0000181294.67212.79 is OK
- 10.3389/fneur.2016.00238 is OK
- 10.1371/journal.pone.0118576 is OK
- 10.3389/fnimg.2022.1031253 is OK
- 10.3174/ajnr.a5162 is OK
- 10.1111/jon.12666 is OK
- 10.1016/j.neuroimage.2016.10.009 is OK
- 10.1097/brs.0000000000001842 is OK
- 10.1002/brb3.797 is OK
- 10.3174/ajnr.a5163 is OK
- 10.1007/s00586-012-2176-4 is OK
- 10.1016/j.neuroimage.2010.07.033 is OK
- 10.1111/jon.12372 is OK
- 10.1212/nxg.0000000000000401 is OK
- 10.1148/radiol.2433060583 is OK
- 10.1002/nbm.3530 is OK
- 10.21037/qims-21-782 is OK
- 10.3389/fneur.2021.637198 is OK
- 10.1097/00007632-199903150-00023 is OK
- 10.1155/2014/719520 is OK
- 10.1038/s41597-022-01571-4 is OK
- 10.5281/zenodo.6511596 is OK
- 10.31219/osf.io/h89js is OK
- 10.1371/journal.pcbi.1009651 is OK
- 10.1371/journal.pcbi.1011230 is OK

MISSING DOIs

- 10.1001/jama.2009.594 may be a valid DOI for title: Gray’s Anatomy: The Anatomical Basis of Clinical Practice

INVALID DOIs

- None
```

**Comment by roboneuro** [comment]
:wave: @openjournals/executable-eics, this paper is ready to be accepted and published.

Check final proof :point_right::page_facing_up: [Download article](https://raw.githubusercontent.com/neurolibre/preprints/neurolibre.00017/neurolibre.00017/10.55458.neurolibre.00017.pdf)

If the paper PDF and the deposit XML files look good in https://github.com/neurolibre/preprints/pull/58, then you can now move forward with accepting the submission by compiling again with the command `@editorialbot accept`

**Comment by agahkarakuzu** [comment]
@roboneuro sync pdf

**Comment by roboneuro** [comment]
I'm sorry human, I don't understand that. You can see what commands I support by typing:

`@roboneuro commands`

**Comment by agahkarakuzu** [comment]
@roboneuro production sync pdf

**Comment by roboneuro** [comment]
:seedling::recycle::page_facing_up: Synced the summary PDF from the [source](https://raw.githubusercontent.com/neurolibre/preprints/neurolibre.00017/neurolibre.00017/10.55458.neurolibre.00017.pdf), should be now available at https://preprint.neurolibre.org/10.55458/neurolibre.00017.pdf?no-cache

**Comment by agahkarakuzu** [comment]
@roboneuro publish

**Comment by roboneuro** [comment]
I'm sorry human, I don't understand that. You can see what commands I support by typing:

`@roboneuro commands`

**Comment by agahkarakuzu** [comment]
@roboneuro accept

**Comment by roboneuro** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by roboneuro** [comment]
**Ensure proper citation** by uploading a plain text CITATION.cff file to the default branch of your repository.

If using GitHub, a *Cite this repository* menu will appear in the About section, containing both APA and BibTeX formats. When exported to Zotero using a browser plugin, Zotero will automatically create an entry using the information contained in the .cff file.

You can copy the contents for your CITATION.cff file here:

<details><summary><strong>CITATION.cff</strong></summary>
<p>

```
cff-version: "1.2.0"
authors:
- family-names: Valošek
  given-names: Jan
  orcid: "https://orcid.org/0000-0002-7398-4990"
- family-names: Bédard
  given-names: Sandrine
  orcid: "https://orcid.org/0000-0001-9859-1133"
- family-names: Keřkovský
  given-names: Miloš
  orcid: "https://orcid.org/0000-0003-0587-9897"
- family-names: Rohan
  given-names: Tomáš
  orcid: "https://orcid.org/0000-0002-7105-583X"
- family-names: Cohen-Adad
  given-names: Julien
  orcid: "https://orcid.org/0000-0003-3662-9532"
doi: 10.55458
message: To reference this work, please cite our reproducible preprint
  in NeuroLibre.
preferred-citation:
  authors:
  - family-names: Valošek
    given-names: Jan
    orcid: "https://orcid.org/0000-0002-7398-4990"
  - family-names: Bédard
    given-names: Sandrine
    orcid: "https://orcid.org/0000-0001-9859-1133"
  - family-names: Keřkovský
    given-names: Miloš
    orcid: "https://orcid.org/0000-0003-0587-9897"
  - family-names: Rohan
    given-names: Tomáš
    orcid: "https://orcid.org/0000-0002-7105-583X"
  - family-names: Cohen-Adad
    given-names: Julien
    orcid: "https://orcid.org/0000-0003-3662-9532"
  date-published: 2023-10-17
  doi: 10.55458/neurolibre.00017
  journal: NeuroLibre Reproducible Preprints
  publisher:
    name: NeuroLibre
  title: A database of the healthy human spinal cord morphometry in the
    PAM50 template space
  type: preprint
  url: "https://neurolibre.org/papers/10.55458/neurolibre.00017"
title: A database of the healthy human spinal cord morphometry in the
  PAM50 template space

```

</p>
</details>

If the repository is not hosted on GitHub, a .cff file can still be uploaded to set your preferred citation. Users will be able to manually copy and paste the citation.

Find more information on .cff files [here](https://github.com/citation-file-format/citation-file-format/blob/main/README.md) and [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files).

**Comment by roboneuro** [comment]
🌰🌱🌺 **THIS IS NOT A DRILL, YOU HAVE JUST PUBLISHED A NEUROLIBRE REPRODUCIBLE PREPRINT!** 🌺🌸🍀

Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/neurolibre/preprints/pull/59
1. Wait a couple of minutes, then verify that the paper DOI resolves [https://doi.org/](https://doi.org/)
2. If everything looks good, then close this review issue.
3. Party like you just published more than a PDF! 🎉🌈🦄💃👻🤘

Any issues? Notify your editorial technical team...

**Comment by agahkarakuzu** [comment]
@valosekj @jcohenadad @sandrinebedard thank you for this great submission, happy to see it on the front page of NeuroLibre! 

You can see that the DOI resolves into the publication page:

https://doi.org/10.55458/neurolibre.00017

Above you can follow instructions to add `cff` to any repo if you would like others to cite this preprint. 

For this submission, I had to edit the [container archive](https://zenodo.org/records/10002182), as repo2data was not called to download the data in the notebook. I updated the description so that people can download the data from zenodo and mount it to the container simply by following those instructions.

As this is a beta release, each submission helps us identify and resolve issues, so thank you for your patience, and looking forward to more submissions! Instead of a PDF, you have 4 DOIs archiving reproducibility assets (including a 2.2GB Docker image), one reproducible preprint with interactive figures , all inter-linked through https://doi.org/10.55458/neurolibre.00017. 

You will be notified by ResearchGate by tomorrow, but it takes some more time for google scholar's web spiders to discover these. Either way, the content is officially registered by Crossref and you can cite it as you see fit.

**Comment by jcohenadad** [comment]
This is so exciting! Thank you @agahkarakuzu and @roboneuro 😊

**Comment by roboneuro** [comment]
![](https://github.com/neurolibre/brand/blob/main/png/neurolibre_main.png?raw=true)

## 🌰 🌱 🌺 Congratulations on your NeuroLibre Reproducible Preprint (NRP)! 🍀 🚀 ♻️ 

By publishing this NRP, you've gone beyond simply posting another traditional preprint, demonstrating your commitment to transparency and reproducibility in presenting your new findings.  This achievement is certainly something to take pride in, and if you wish to showcase it in your repository:

```
Markdown:
[![DOI](https://neurolibre.org/papers/10.55458/neurolibre.017/status.svg)](https://doi.org/10.55458/neurolibre.017)

HTML:
<a style="border-width:0" href="https://doi.org/10.55458/neurolibre.017">
  <img src="https://neurolibre.org/papers/10.55458/neurolibre.017/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://neurolibre.org/papers/10.55458/neurolibre.017/status.svg
   :target: https://doi.org/10.55458/neurolibre.017
```
This is how it will look in your, e.g., README:

[![DOI](https://neurolibre.org/papers/10.55458/neurolibre.017/status.svg)](https://doi.org/10.55458/neurolibre.017)

**We need your help!**

NeuroLibre is a community-run reproducible preprint server and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to become a technical screener
- Contribute to the development

For details, please visit our [documentation](https://docs.neurolibre.org)

**Comment by valosekj** [comment]
Thank you @agahkarakuzu! 💯

**Comment by valosekj** [comment]
> ```
> Markdown:
> [![DOI](https://neurolibre.org/papers/10.55458/neurolibre.017/status.svg)](https://doi.org/10.55458/neurolibre.017)
> 
> HTML:
> <a style="border-width:0" href="https://doi.org/10.55458/neurolibre.017">
>   <img src="https://neurolibre.org/papers/10.55458/neurolibre.017/status.svg" alt="DOI badge" >
> </a>
> 
> reStructuredText:
> .. image:: https://neurolibre.org/papers/10.55458/neurolibre.017/status.svg
>    :target: https://doi.org/10.55458/neurolibre.017
> ```

Just a nitpick; the DOI in the above-mentioned code section should be `https://doi.org/10.55458/neurolibre.00017` (i.e., with two extra leading zeros). When adding these two zeros, the badge is working!

**Comment by sandrinebedard** [comment]
Thank you @agahkarakuzu!
